### ISSUE TITLE
javascript not working on school chromebook

### ISSUE THREAD

**Issue by twinturboq** [JavaScythe/bnkr-pro#12]
What do I do?

**Comment by JavaScythe** [comment]
by javascript, i think you mean bookmarklets, right?

**Comment by twinturboq** [comment]
Yes

**Comment by twinturboq** [comment]
and when you use "javascript:-insert random script here-"

**Comment by twinturboq** [comment]
I don't understand?

**Comment by twinturboq** [comment]
Wait, so I can't use my bookmarks anymore? because all the bookmarks I got have JavaScript on it, so what does that mean?

**Comment by JavaScythe** [comment]
if you want we can continue in discord: "[object Object]#3827"

??? you just told me you can't use bookmarklets?

**Comment by twinturboq** [comment]
Yes, I did say I can't use bookmarks ANYMORE but it's because JavaScript isn't working on my Chromebook.

**Comment by JavaScythe** [comment]
???
Javascript as a whole is disabled on your chromebook???? that is highly unlikely.

**Comment by twinturboq** [comment]
Oh yes, it is and I have proof.

**Comment by JavaScythe** [comment]
so 98% of websites don't work?
for example, a school portal doesn't work, google classroom doesnt work?

**Comment by twinturboq** [comment]
I'll send you the proof, and I cannot use bookmarks, I can't use scripts on java at all. 
[Screen recording 2023-03-02 1.51.21 PM.webm](https://user-images.githubusercontent.com/91972887/222524470-d4ee308d-7178-4dac-83da-233cca1c3ce6.webm)

**Comment by JavaScythe** [comment]
that isn't javascript, those are bookmarks
more specifically, bookmarklets, which use the "javscript:" protocol to execute JS
so, javascript is enabled on your computer, but bookmarks aren't
bunker will likely work on your chromebook, but if you wanna make sure go to "chrome://policy", export to JSON and ill check it

**Comment by twinturboq** [comment]
https://drive.google.com/file/d/10bf1_KMA-jSgngXj-5MZGgF8ALOH5xhm/view?usp=sharing
If it doesn't work, lmk.

**Comment by JavaScythe** [comment]
access denied, asked for perms from "slashmod6@gmail.com"

**Comment by twinturboq** [comment]
It should be sent.

**Comment by twinturboq** [comment]
Also discord is blocked for me unfortunately, so I cannot contact you.

**Comment by JavaScythe** [comment]
np. your chromebook will be just fine for bunker pro.
see you 3/14 !

**Comment by twinturboq** [comment]
I really hope it does. So what do I do now?

**Comment by JavaScythe** [comment]
wait till release on march 13

**Comment by twinturboq** [comment]
Damn, that long? Any way to remove GoGuardian by any chance while I wait?

**Comment by JavaScythe** [comment]
bruh
you can try bunker v5 at https://github.com/JavaScythe/bunker

**Comment by twinturboq** [comment]
I'm sorry.
